### PR TITLE
adapt geolocation to new structure

### PR DESCRIPTION
**Pull request by Co0oder**

### CODE REVIEW

**Comment by umilanovich** [comment]
Please resolve merge conflict

File: src/app/features/home/map/choose-place-dialog/choose-place-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+@Component({
+  selector: 'app-choose-place-popup',
+  templateUrl: './choose-place-dialog.component.html',
+  styleUrls: ['./choose-place-dialog.component.scss'],
+})
+export class ChoosePlaceDialogComponent implements OnInit {
+  public geolocationEnabled = false;
+  public offices: IOffice[];
+  public selectedOffice = 0;
+  public selectedPlaceDetection = '1';
```
Comment (umilanovich) [inline src/app/features/home/map/choose-place-dialog/choose-place-dialog.component.ts:18]:
```suggestion
  public selectedPlaceDetection: placeDetectionMethod = placeDetectionMethod.officeLocation;
```

File: src/app/core/enums/place-detection-method.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export enum placeDetectionMethod {
```
Comment (umilanovich) [inline src/app/core/enums/place-detection-method.ts:1]: To indicate that this is enum you can name it `PlaceDetectionMethod`, also there is no need to asign it a number/string https://www.typescriptlang.org/docs/handbook/enums.html

File: src/app/features/home/map/choose-place-dialog/choose-place-dialog.component.ts (hunk 1 above)
Comment (Co0oder) [reply src/app/features/home/map/choose-place-dialog/choose-place-dialog.component.ts]: changed

File: src/app/core/enums/place-detection-method.ts (hunk 2 above)
Comment (Co0oder) [reply src/app/core/enums/place-detection-method.ts]: changed

**Review by umilanovich — APPROVED** [review]

**Pull request merged**